src/Files.OpenDialog/FilesLauncher/FilesLauncher.cpp

```diff
@@ -0,0 +1,195 @@
+//
+
+#include <iostream>
+
+#pragma comment(lib, "ole32.lib")
+#pragma comment(lib, "Propsys.lib")
+#pragma comment(lib, "user32.lib")
+#pragma comment(lib, "uuid.lib")
+
+#include <wil/resource.h>
+
+#include "OpenInFolder.h"
+
+#define ID_TIMEREXPIRED 101
+
+LRESULT CALLBACK WindowProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam);
+
+int WINAPI WinMain(HINSTANCE hInstance,
+	HINSTANCE hPrevInstance,
+	LPSTR     lpCmdLine,
+	int       cmdShow)
+{
+	auto oleCleanup = wil::OleInitialize_failfast();
+
+	FILE* _debugStream = NULL;
+	PWSTR pszPath = NULL;
+	HRESULT hr = SHGetKnownFolderPath(FOLDERID_Desktop, 0, NULL, &pszPath);
+	if (SUCCEEDED(hr))
+	{
+		TCHAR debugPath[MAX_PATH];
+		wsprintf(debugPath, L"%s\\%s", pszPath, L"open_in_folder.txt");
+		//_wfreopen_s(&_debugStream, debugPath, L"w", stdout);
```

Comment:
@yaichenbaum try to uncomment this line, deploying and switching off-on the toggle. It should create a debug log on desktop. Could you post it's contents after you try to invoke explorer with Files uninstalled?

Reply:
Also please verify that this file does no longer exist when Files is uninstalled (you may have multiple release of Files) -> %LOCALAPPDATA%\\Microsoft\\WindowsApps\\files.exe

Reply:
> Also please verify that this file does no longer exist when Files is uninstalled (you may have multiple releases of Files) -> %LOCALAPPDATA%\Microsoft\WindowsApps\files.exe

The file is removed on uninstall.

> What does it happen when you try and the call is not forwarded to explorer? You get the usual error message? Does it even work for you when Files is installed?

I get an error "Application not found".

Reply:
> @yaichenbaum try to uncomment this line, deploying and switching off-on the toggle. It should create a debug log on desktop. Could you post its contents after you try to invoke explorer with Files uninstalled?

It's not creating a debug file, is it posible the launcher isn't properly deplying?

Reply:
The launcher should be copied to %LocalAppData%\Files\FilesLaucher.exe, does it exist? Does Files launch when it's installed?

Reply:
The launcher is being copied and it's creating the log file now but nothing is being entered into it.

Reply:
[f5a10e1](https://github.com/files-community/Files/pull/8684/commits/f5a10e1e858e739ba7d165789f15cc25fc867a64) should fix the issue.